form-validation.js

```diff
@@ -7,25 +7,29 @@ const messageMobile = document.querySelector('#messageEmail-mobile');
 const nameInput = document.querySelector('.name');
 const emailInput = document.querySelector('.email');
 const messageInput = document.querySelector('.message');
-// const formbutton = document.querySelector('#form-button');
-// const formbuttondesktop = document.querySelector('#form-button-desk');
+const rawData = localStorage.getItem('cachedFormData');
 
 const formobject = {
   name: '',
   email: '',
   message: '',
 };
 
-function storeInput() {
-  const stringobject = JSON.stringify(formobject);
-  nameInput.value = formobject.name;
+if (rawData) {
+  const formobject = JSON.parse(rawData);
   emailInput.value = formobject.email;
+  nameInput.value = formobject.name;
   messageInput.value = formobject.message;
-  localStorage.setItem('formobject', stringobject);
 }
 
-form.addEventListener('click', storeInput());
-mobileForm.addEventListener('click', storeInput());
+const dataStore = (event) => {
```

Comment:
- [ ] Please also make sure that the users preserved data are populated in the input once the page loads if there are any data. :+1: :smile: